sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -222,6 +240,18 @@ private void initializeSkin(Application app, OoyalaSkinLayout l, OoyalaPlayer p,
                     FrameLayout.LayoutParams.MATCH_PARENT);
     l.addView(rootView, frameLP);
     rootView.setBackgroundColor(Color.TRANSPARENT);
+
+    boolean isTargetTV = isTargetDeviceTV(_layout.getContext());
```

Comment:
create class variable instead of this. And in function isStereoSupported change check UI_MODE_SERVICE to this boolean